### PR TITLE
Refs [TASK][34319]  Handle get data standing 

### PR DESCRIPTION
**Pull request by khangvv158**
## Related Tickets
- [#Task 34319: Handle get data standing](https://edu-redmine.sun-asterisk.vn/issues/34319)
## WHAT
- Create Model, get data standing
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)

### CODE REVIEW

File: app/src/main/java/com/sun/uefascore/data/source/remote/StandingRemoteDataSource.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    }
+
+    override fun <T> getDataStandingLeague(listener: OnFetchDataJsonListener<T>) {
+        val stringUrl = Constant.BASE_URL + STANDINGS + Constant.BASE_SEASON + Constant.BASE_LEAGUE
+        GetJsonFromUrl(listener, TypeModel.STANDING_LEAGUE).execute(stringUrl)
+    }
+
+    companion object {
+        val instance by lazy { Holder.INSTANCE }
+        private const val STANDINGS = "standings?"
```
Comment (huytq-2422) [inline app/src/main/java/com/sun/uefascore/data/source/remote/StandingRemoteDataSource.kt]: thứ tự thuộc tính 
const 
override
val 
var

File: app/src/main/java/com/sun/uefascore/data/source/remote/StandingRemoteDataSource.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package com.sun.uefascore.data.source.remote
+
+import com.sun.uefascore.data.source.StandingDataSource
+import com.sun.uefascore.data.source.remote.fetchjson.GetJsonFromUrl
+import com.sun.uefascore.utils.Constant
+import com.sun.uefascore.utils.TypeModel
+
+class StandingRemoteDataSource : StandingDataSource.Remote {
```
Comment (huytq-2422) [inline app/src/main/java/com/sun/uefascore/data/source/remote/StandingRemoteDataSource.kt]: lỗi của bạn Lành hôm qua

File: app/src/main/java/com/sun/uefascore/data/source/remote/fetchjson/ParseDataWithJson.kt [hunk 3]
```diff
@@ -43,6 +46,23 @@ class ParseDataWithJson {
     fun parseJson(jsonString: String, typeModel: TypeModel): Any? =
             try {
                 when (typeModel) {
+                    TypeModel.STANDING_LEAGUE -> {
+                        parseJsonToObject(
+                            JSONObject(jsonString).getJSONArray(
+                                Constant.RESPONSE
+                            ).getJSONObject(0).getJSONObject(StandingLeagueEntry.LEAGUE), typeModel
```
Comment (huytq-2422) [inline app/src/main/java/com/sun/uefascore/data/source/remote/fetchjson/ParseDataWithJson.kt]:
```suggestion
                            JSONObject(jsonString).getJSONArray(Constant.RESPONSE)
                                .getJSONObject(0)
                                .getJSONObject(StandingLeagueEntry.LEAGUE), 
                            typeModel
```

File: app/src/main/java/com/sun/uefascore/data/source/remote/fetchjson/ParseDataWithJson.kt [hunk 4]
```diff
@@ -43,6 +46,23 @@ class ParseDataWithJson {
...
+                        )
+                    }
+                    TypeModel.STANDING_GROUP -> {
+                        parseJsonToListJson(
+                            JSONObject(jsonString).getJSONArray(
+                                StandingGroupEntry.STANDING_GROUP
+                            ), typeModel
```
Comment (huytq-2422) [inline app/src/main/java/com/sun/uefascore/data/source/remote/fetchjson/ParseDataWithJson.kt]: tương tự

File: app/src/main/java/com/sun/uefascore/data/source/remote/StandingRemoteDataSource.kt (hunk 1 above)
Comment (khangvv158) [reply app/src/main/java/com/sun/uefascore/data/source/remote/StandingRemoteDataSource.kt]: fixed

File: app/src/main/java/com/sun/uefascore/data/source/remote/StandingRemoteDataSource.kt (hunk 2 above)
Comment (khangvv158) [reply app/src/main/java/com/sun/uefascore/data/source/remote/StandingRemoteDataSource.kt]: fixed

File: app/src/main/java/com/sun/uefascore/data/source/remote/fetchjson/ParseDataWithJson.kt (hunk 3 above)
Comment (khangvv158) [reply app/src/main/java/com/sun/uefascore/data/source/remote/fetchjson/ParseDataWithJson.kt]: fixed

**Review by huytq-2422 — APPROVED** [review]

**Pull request merged**